.travis.yml

```diff
@@ -6,7 +6,9 @@ addons:
   ssh_known_hosts: api.openfisca.fr
 install:
   - pip install --upgrade pip wheel  # pip >= 8.0 needed to be compatible with "manylinux" wheels, used by numpy >= 1.11
-  - pip install "OpenFisca-France==18.5.3"  # Needed only for tests.
+  - pip install --editable git+https://github.com/openfisca/openfisca-core.git@params-yaml#egg=OpenFisca-Core
```

Comment:
Commit is called "[Read before merge] Travis configuration", not explicit enought.